scripts/build-rss.js

```diff
@@ -1,22 +1,69 @@
 import fs from 'fs'
-import RSS from 'rss'
-import getAllPostPreviews from '../src/getAllPostPreviews'
+import path from 'path'
+import ReactDOMServer from 'react-dom/server'
+import { MDXProvider } from '@mdx-js/react'
+import { Feed } from 'feed'
 
-const feed = new RSS({
+import { mdxComponents } from '../src/components/Post'
+import { getAllPosts } from '../src/getAllPostPreviews'
+
+const siteUrl = 'https://blog.tailwindcss.com'
+
+const feed = new Feed({
   title: 'Blog – Tailwind CSS',
-  site_url: 'https://blog.tailwindcss.com',
-  feed_url: 'https://blog.tailwindcss.com/feed.xml',
+  description: 'All the latest Tailwind CSS news, straight from the team.',
+  id: siteUrl,
+  link: siteUrl,
+  language: 'en',
+  image: `${siteUrl}/favicon-32x32.png`,
+  favicon: `${siteUrl}/favicon.ico`,
+  copyright: 'All rights reserved 2020, Tailwind Labs',
+  feedLinks: {
+    json: `${siteUrl}/json`,
+    atom: `${siteUrl}/atom`,
```

Comment:
Am I right in thinking that this should be `/feed.json` and `/atom.xml`?

Reply:
It can be that as well 👍